lib/Common/PlatformAgnostic/Numbers.h

```diff
@@ -0,0 +1,55 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#ifndef RUNTIME_PLATFORM_AGNOSTIC_COMMON_NUMBERS
+#define RUNTIME_PLATFORM_AGNOSTIC_COMMON_NUMBERS
+
+namespace PlatformAgnostic
+{
+namespace Numbers
+{
+    class Utility
```

Comment:
Just a nit- why have this Utility class? PlatformAgnostic::Numbers::NumberToDefaultLocaleString seems fine